.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on: [
+    push,
+    workflow_dispatch
+]
+
+jobs:
```

Comment:
* This flow does nothing for now.
* Use this https://github.com/spryker-eco/adyen/pull/38 as an example.